docs/installation/formatter-configuration.md

```diff
@@ -0,0 +1,56 @@
+# Formatter Configuration
+
+There are two ways to configure [Reqnroll Formatters](../reporting/reqnroll-formatters.md).
+
+* [](#configuration-file)
+* [](#environment-variables)
+
+## Defaults
+
+Unless overwritten by using the Reqnroll configuration file and/or environment variable, Reqnroll will use the following defaults to configure formatters.
+
+```{list-table}
+:header-rows: 1
+
+* - Setting
+  - Value
+  - Description
+* - Enabled
+  - true/false
+  - Controls whether Reqnroll formatters will be enabled during test execution. <br/> *Default:* `true`
+```
+
+## Configuration File
+
+The `formatters` section of the [`reqnroll.json` configuration file](configuration.md) can be used to configure formatters. Each section within `formatters` enables and configures a built-in or custom formatter. You can enable multiple formatters.
+
+The following example enables both the [HTML](../reporting/reqnroll-formatters.md#html-formatter) and the [Cucumber Messages](../reporting/reqnroll-formatters.md#cucumber-messages-formatter) formatter with custom output file paths.
+
+```{code-block} json
+:caption: reqnroll.json
+{
+  "$schema": "https://schemas.reqnroll.net/reqnroll-config-latest.json",
+
+    "bindingAssemblies": [
+    ],
+    "formatters": {
+        "html" : { "outputFilePath" : "report\\living_doc.html" },
+        "message" : { "outputFilePath" : "report\\cucumber_messages.ndjson" }
+    }
+}
+```
+
+If the formatter section is omitted or empty, the report for that particular formatter is not enabled.
```

Comment:
... or empty .. won't be true for very long, right?

Reply:
I removed the empty part, to make it simpler